api/storage_service.cc

```diff
@@ -801,6 +801,11 @@ void set_storage_service(http_context& ctx, routes& r, sharded<service::storage_
         });
     });
 
+    register_void_routine(ctx, "move", [&ss] (std::unordered_map<sstring, sstring>&& val) {
+        auto new_token = std::move(val["new_token"]);
+        return ss.local().move(std::move(new_token));
+    });
```

Comment:
move is a completely generic name, we should have a more specific name (especially since moving tokens is totally unused). It's probably easier to drop the patch than think of a good name.